addons/project/models/project.py

```diff
@@ -245,9 +245,19 @@ def _compute_task_count(self):
         for data in task_data:
             result_wo_subtask[data['project_id'][0]] += data['display_project_id']
             result_with_subtasks[data['project_id'][0]] += data['project_id_count']
+        task_all_data = self.env['project.task'].with_context(active_test=False)._read_group(
+            [('project_id', 'in', self.ids),
+             ('is_closed', '=', False)],
+            ['project_id', 'display_project_id:count'], ['project_id'])
```

Comment:
We could maybe isolate the domain, fields and group by in variables that could be used both read_group. That way we are reducing code maintenance and we could also probably write the read_groups in one line.
What do you think?

Reply:
@IT-Ideas Done the changes. Thanks.

Reply:
@hiroh-odoo , why can't we filter the projects based on `active/inactive` and based on conditions we can do the separate `_read_group` for both instead of calling the read_group each(two) time even if inactive/active records do not exist?
@IT-Ideas , what do you suggest? 

Thanks